### PR TITLE
create EUREC4A-MIP entry

### PR DESCRIPTION
**Pull request by observingClouds**
This PR adds a section about the EUREC4A-MIP effort on how to access boundary conditions, how to provide new model runs and how to access any available simulations.

### CODE REVIEW

File: how_to_eurec4a/eurec4a_mip.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# EUREC⁴A-MIP
+
+The EUREC⁴A-Model intercomparison project (EUREC⁴A-MIP) is an ongoing effort to test the ability of a variety of simulations to capture the observed variability of the trade-winds.
```
Comment (observingClouds) [inline how_to_eurec4a/eurec4a_mip.md:16]: @torresalavez could you fill in here and below some additional information? That would be great!

Comment (observingClouds) [reply how_to_eurec4a/eurec4a_mip.md:16]: @torresalavez I'll merge this branch whenever the review is successful, but please open a new PR and add some additional information about the boundary conditions in particular those of the PGW. How were they calculated, what are important references, how shall they be used (e.g. are they differences and need to be added to the boundary conditions etc.). Your input would greatly improve this section!

**Review by RobertPincus — APPROVED** [review]
@observingClouds This looks neat! I'm looking forward to seeing results. @torresalavez please do add further documentation.

**Review by RobertPincus — APPROVED** [review]
@observingClouds This looks neat! I'm looking forward to seeing results. @torresalavez please do add further documentation.

**Pull request merged**